### ISSUE TITLE
Corruption: block checksum mismatch

### ISSUE THREAD

**Issue by ScottMcDermid** [bitcoin-sv/bitcoin-sv#294]
### Describe the issue

We're attempting to do a fresh sync, but `bitcoind` freezes after 2-3 hours claiming it encountered:

```
Corruption: block checksum mismatch: /bitcoin-sv/data/blocks/index/000211.ldb
```

#### If so, please list the steps to reproduce below:

1. Nuke the contents of `data/*`
2. Run `bitcoind` in an attempt to sync the blockchain

### Expected behaviour
It should sync the entire blockchain.

### Actual behaviour
It stalls after 2-3 hours.

### What version of bitcoin-sv are you using?
1.0.14

### Machine specs:
- OS: AlmaLinux 8.7 (Stone Smilodon)
- CPU: AMD Ryzen 7 5800X 8-Core Processor
- RAM: 64GiB DDR4 2666MHz
- Disk size: 20TB
- Disk Type (HD/SDD): HD

### Any extra information that might be useful in the debugging process.

`bitcoind.log`: https://pastebin.com/tYzY1qkL
`livenet.conf`: https://pastebin.com/LFTqqUar

**Comment by richmills3** [comment]
Hi

From the error you report that seems to be a file-system corruption problem down inside the leveldb library, which is surprising.

I can't access the config file you posted to pastebin, it's giving me a 404. Could you try to provide it again please?

A couple of other questions:

1. Is the machine in question a physical box or a virtual machine?
2. How reproducible is this? Does it fail every time or was this seen just once?
3. If it fails every time, have you tried an older version of bitcoind (say 1.0.13) to see if it still fails there?

**Comment by ScottMcDermid** [comment]
Hi there, thanks for the quick response!

> I can't access the config file you posted to pastebin, it's giving me a 404. Could you try to provide it again please?

Pastebin seems to have removed it after pending moderation. It's small enough, that I can attach the file itself.

[livenet.conf.zip](https://github.com/bitcoin-sv/bitcoin-sv/files/11203167/livenet.conf.zip)

> Is the machine in question a physical box or a virtual machine?

It's a physical box.

> How reproducible is this? Does it fail every time or was this seen just once?

It consistently fails to sync. We've tried a number of times without luck.

> If it fails every time, have you tried an older version of bitcoind (say 1.0.13) to see if it still fails there?

Yes, we've tried 1.0.11 and 1.0.13. It still seemed to fail after a few hours with a similar corruption error. I actually still have the 1.0.11 log: https://pastebin.com/Y3UCgk3L

For more context, we had a 1.0.11 fullnode fully synced until about 2 months ago. Since then, we've struggled to re-sync after attempting a `reindex`.
